x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/cti/threat_intel_source/query.threat_intel_source.dsl.test.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { buildTiDataSourceQuery } from './query.threat_intel_source.dsl';
+import { mockOptions, expectedDsl } from './__mocks__';
```

Comment:
These `__mocks__` files are a relic of jest auto-mocking that we aren't using, here. I would just name that file `index.mock.ts` instead (or e.g. `query.threat_intel_source.dsl.mock.ts` if that's more accurate)

Reply:
Just move the mock value, exactly to the tests